docs/design/metrics.md

```diff
@@ -1,39 +1,40 @@
 ## Scheduler Monitoring
 
 ## Introduction
-Currently users can leverage controller logs and job events to monitor scheduler. While useful for debugging, none of this options is particularly practical for monitoring kube-batch behaviour over time. There's also requirement like to monitor kube-batch in one view to resolve critical performance issue in time [#427](https://github.com/kubernetes-sigs/kube-batch/issues/427).
+Currently users can leverage controller logs and job events to monitor scheduler. While useful for debugging, none of this options is particularly practical for monitoring volcano behaviour over time. There's also requirement like to monitor volcano in one view to resolve critical performance issue in time [#427](https://github.com/kubernetes-sigs/volcano/issues/427).
```

Comment:
Seems the issue link need not to be changed.

Reply:
Yes, I've already fix it. In addition, I add more metrics defination.